git.adoc

```diff
@@ -0,0 +1,211 @@
+= Git guidelines
+Jakub Jirutka
+// URIs
+:uri-git-scm: https://git-scm.com
+
+This guide aims to provide the ground rules and recommendations for using {uri-git-scm}[git] in software projects with releases.
+
+The described rules are based on widely recognized best-practices for using git (see <<References, references>>).
+The workflow is based on https://docs.gitlab.com/ee/workflow/gitlab_flow.html[GitLab Flow].
+
+To keep it short and easy to follow, it does not explain reasons for particular rules and alternative approaches.
+If you’re in doubt, consult articles in the <<References>> section — they explain most of the concepts into a great detail.
+
+
+== General
```

Comment:
There should be recommended non-default global/general git setting chapter somewhere - e.g. cr/lf handling on MSWin platforms